From d376bd3f4d1e6bd8b6c7d4151caa701b238235ac Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Tue, 14 Dec 2021 12:58:39 -0600
Subject: [PATCH] Platform.h: tie typedef for HRESULT to !windows

The existence of `__clang__` there breaks clang on Windows compilation
due to conflicting LONG vs INT typedefs

Signed-off-by: Christopher Degawa <ccom@randomderp.com>
---
 amf/public/include/core/Platform.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/amf/public/include/core/Platform.h b/amf/public/include/core/Platform.h
index 7fa61cc..a175e62 100644
--- a/amf/public/include/core/Platform.h
+++ b/amf/public/include/core/Platform.h
@@ -72,7 +72,7 @@
 //     #error Need to define AMF_ALIGN
  #endif
 
-#if defined(__linux) || (__clang__)
+#if !defined(_WIN32)
 typedef signed int HRESULT;
 #define SUCCEEDED(hr) (((HRESULT)(hr)) >= 0)
 #define FAILED(hr) (((HRESULT)(hr)) < 0)
